e2e/support/config.js

```diff
@@ -168,7 +169,7 @@ const defaultConfig = {
   // New `specPattern` is the combination of the old:
   //   1. testFiles and
   //   2. integrationFolder
-  specPattern: "e2e/test/**/*.cy.spec.{js,ts}",
+  specPattern: "e2e/test/scenarios/**/*.cy.spec.{js,ts}",
```

Comment:
Sample Apps e2e tests are stored in a folder under `e2e/test` path, and we should not run them during regular e2e suite